Automattic/jetpack
# Search: Improve a11y via headings hierarchy and aria roles

Fixes #22156. cc @jpumfreyamergin, who first reported this issue.

#### Changes proposed in this Pull Request:
* Reorganizes the headings hierarchy for improved accessibility.
* Reorganizes and adds aria roles for improved accessibility. 

#### Jetpack product discussion
None.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your site with a Jetpack Search subscription.
* Perform a site search.
* Ensure that the elements have reasonable aria roles and heading hierarchies.
